src/scicookie/{{cookiecutter.project_slug}}/tests/test_{{cookiecutter.project_slug}}.py

```diff
@@ -1,4 +1,5 @@
 """Tests for `{{ cookiecutter.project_slug }}` package."""
+{%- if cookiecutter.use_pytest == "yes" -%}
```

Comment:
could you revisit this file again?

keep in mind that it would be possible to have both pytest and hypothesis.
also please, check the result locally to see what is the final result for this file for all the 3 possibilities (pytest, hypothesis, pytest + hypothesis).

also pay attention on the jinja2 modifications %-/-%, %+/+% or just % .. and the combinations among them